From 9ed10a28e66883764d9d45b4624cafe62b9d5054 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 8 Jan 2020 13:19:03 +0100
Subject: [PATCH 2/2] Revert "ubus: update to version 2019-12-27"

With libubox, it needs to be reverted as well. (86183e5b6b)
Fosquitto does not start and it is communicating with ubus and libubox

This reverts commit 64c45d95d622bf9f3727843fe6e05de71d310a9c.
---
 package/system/ubus/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/package/system/ubus/Makefile b/package/system/ubus/Makefile
index d3fe452252..6653132351 100644
--- a/package/system/ubus/Makefile
+++ b/package/system/ubus/Makefile
@@ -5,9 +5,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/ubus.git
-PKG_SOURCE_DATE:=2019-12-27
-PKG_SOURCE_VERSION:=041c9d1c052bb4936fd03240f7d0dd64aedda972
-PKG_MIRROR_HASH:=7febbe78dd178d8d7ccacff54c00fb2d19a53624005920db8cd85236f54f8324
+PKG_SOURCE_DATE:=2018-10-06
+PKG_SOURCE_VERSION:=221ce7e7ff1bd1a0c9995fa9d32f58e865f7207f
+PKG_MIRROR_HASH:=e6ab3f009688740eb6d7a1857df59a6ba10ae8ae0fac78ae0fd6672f71c02dce
 CMAKE_INSTALL:=1
 
 PKG_LICENSE:=LGPL-2.1
@@ -36,7 +36,7 @@ define Package/libubus
   SECTION:=libs
   CATEGORY:=Libraries
   DEPENDS:=+libubox
-  ABI_VERSION:=20191227
+  ABI_VERSION:=20170705
   TITLE:=OpenWrt RPC client library
 endef
 
-- 
2.24.1

